### PR TITLE
Add support for VF type interface rate limiting

### PR DESCRIPTION
**Pull request by gaolitao**
Send out for review feedback, will update Gopkg.lock after https://github.com/Wind-River/gophercloud/pull/8 merged.

---------------------------------------------------------------

Rate limiting setting for VF type interfaces support
has been added in single nic sharing feature in starlingx
https://storyboard.openstack.org/#!/story/2008470

Also need to modify DM code to support this configuraiton.

Signed-off-by: Litao Gao <litao.gao@windriver.com>

### CODE REVIEW

**Review by donpenney — COMMENTED** [review]
LGTM. As you've noted, once the gophercloud update is merged, you can update the reference and we can merge this one (once we've got at least one more reviewer)

**Review by donpenney — APPROVED** [review]

**Review by albailey — APPROVED** [review]

**Pull request merged**
